### DISCUSSION TITLE
When to build an original and portable plot. What to do . What is the original plot and the portable plot? how can made it

### DISCUSSION THREAD

**Discussion by pouryarafieerakii** [Chia-Network/chia-blockchain#5702] [Support]
Are the plots we build in Chia wallet the original plots?
And can Connect or move  them to the pool or not?
What is the original plot and the portable plot?
How to Build them?
If we want to connect to the original Chia pool, We have to start plotting on May 31st?
 if yes? We must stop building the plot now, 
If want to connect to the original Chia pool. 
And there we start plotting again.

**Comment by BrandtH22** [comment]
@pouryarafieerakii , since this ticket was opened a number of years ago without further comments it seems this issue has been resolved. 
We will be closing this ticket but if we have closed this ticket in error do not hesitate to reach out to us again with any followup questions or comments.

To answer your questions:
- OG, portable (plotNFT), or compressed plots can be created in the GUI.
- Any portable plots can be moved across pools (compressed plots are generally portable plots)
- OG plots use the pool public key while portable plots use the pool contract address. The contract is what enables portable plots to move between pools and self farming OG plots cannot be moved between pools.
- One can use the GUI following our plotting guide: https://docs.chia.net/farming-guide/#create-a-plot
- There is no deadline for plotting 


The best place to reach our support team is on Discord (https://discord.gg/chia) or by reopening this ticket.
